### DISCUSSION TITLE
Change in functionality for the RecursiveDepth parameter

### DISCUSSION THREAD

**Discussion by PeterOliverDev** [soenneker/soenneker.utils.autobogus#268] [General]
Hello, our team was using Autobus (the abandoned project) and we've just come across your library.

In the previous library, we used the RecursiveDepth parameter, setting it to 0 to indicate that we didn't want it to be recursive, and it worked fine.

In this new library, when the RecursiveDepth parameter is set to 0, we encounter the exception: Stack empty.

What we wanted to know is whether this is expected behavior, and if so, why the change from 0 to 1?

Thank you!

**Comment by soenneker** [comment]
@PeterOliverDev could you give latest a try. I believe I've addressed this. Thanks

**Reply by PeterOliverDev** [reply]
thank you very much, it works.
